.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,17 +6,21 @@ Closes <!--- Insert Issue Number(s) this PR addresses. Start by typing # will op
 
 ### All Submissions:
 
-* [ ] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
 * [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
 * [ ] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.
 
+### Contribution Checklist
```

Comment:
Should we note that both authors and reviewers should verify this section?

Reply:
Yeah I'm confused about who the intended audience is here. How does it differ from the author checklist?